### ISSUE TITLE
Enhancement - Add calendar to Profile

### ISSUE THREAD

**Issue by EdiWeeks** [damus-io/damus#598]
Add calendar to Profile. If there is an event/live video where user wants to attend, can be scheduled and displayed in calendar. 
Profile
Blocked
Relays
Settings
What's new?
Saved Posts
Draft Posts
Events
**Calendar**

**Comment by alltheseas** [comment]
@tyiu added the calendar NIP https://github.com/nostr-protocol/nips/blob/master/52.md

Now there are meetup-like apps in Flockstr

**Comment by alltheseas** [comment]
Bounty https://bountsr.org/meetup-replacement/

**Comment by jb55** [comment]
On Fri, Nov 10, 2023 at 08:03:16AM -0800, alltheseas wrote:
>@tyiu added the calendar NIP https://github.com/nostr-protocol/nips/blob/master/52.md
>
>Now there are meetup-like apps in Flockstr

not entirely sure I want to turn damus into meetup, but might be
interesting to at least handle the note type.

**Comment by alltheseas** [comment]
> On Fri, Nov 10, 2023 at 08:03:16AM -0800, alltheseas wrote:
> >@tyiu added the calendar NIP https://github.com/nostr-protocol/nips/blob/master/52.md
> >
> >Now there are meetup-like apps in Flockstr
> 
> not entirely sure I want to turn damus into meetup, but might be
> interesting to at least handle the note type.
> 

Agree. One insight that hit home for me is that if Damus is to win, there will be many, many complementary nostr apps. This means that Damus should keep in mind interoperability. Reading different event types is a great start.

[Nip-24](https://github.com/nostr-protocol/nips/blob/master/24.md) extra metadata, and [Nip-89](https://github.com/nostr-protocol/nips/blob/master/89.md) recommended application handlers would be good to discuss as well.

**Comment by alltheseas** [comment]
> > On Fri, Nov 10, 2023 at 08:03:16AM -0800, alltheseas wrote:
> > >@tyiu added the calendar NIP https://github.com/nostr-protocol/nips/blob/master/52.md
> > >
> > >Now there are meetup-like apps in Flockstr
> > 
> > not entirely sure I want to turn damus into meetup, but might be
> > interesting to at least handle the note type.
> > 
> 
> Agree. One insight that hit home for me is that if Damus is to win, there will be many, many complementary nostr apps. This means that Damus should keep in mind interoperability. Reading different event types is a great start.
> 
> [Nip-24](https://github.com/nostr-protocol/nips/blob/master/24.md) extra metadata, and [Nip-89](https://github.com/nostr-protocol/nips/blob/master/89.md) recommended application handlers would be good to discuss as well. 

In the today interoperability issues, we have the long form notes replies appearing as stand alone https://github.com/damus-io/damus/issues/1603

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
